misk/src/main/kotlin/misk/web/jetty/WebActionsServlet.kt

```diff
@@ -59,6 +59,13 @@ internal class WebActionsServlet @Inject constructor(
     }
   }
 
+  override fun service(request: HttpServletRequest?, response: HttpServletResponse?) {
+    if (request?.method == "PATCH" && response != null) {
+      doPatch(request, response)
```

Comment:
You need a `return` here

Reply:
I'm actually confused how the test passes, it should be calling `doPatch` and then end up throwing later in the function. Will debug locally to make sure something else isn't happening.

Reply:
Yeah Jetty throws this in the background but the tests happily passes
```
java.lang.IllegalStateException: COMPLETED
	at org.eclipse.jetty.server.HttpChannelState.sendError(HttpChannelState.java:893)
	at org.eclipse.jetty.server.Response.sendError(Response.java:461)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:732)
	at org.eclipse.jetty.websocket.servlet.WebSocketServlet.service(WebSocketServlet.java:176)
	at misk.web.jetty.WebActionsServlet.service(WebActionsServlet.kt:66)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:755)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:547)
```

Adding the `return` fixes